docs/source/quickstart/model_config.md

```diff
@@ -27,4 +35,13 @@ layers:
     name: final_dense
     units: 1
     activation: null
-```
\ No newline at end of file
+optimizer: 
+  key: adam
+  gradient_clip_value: 5.0
+lr_schedule:
```

Comment:
@mohazahran It will be useful to add a description of each of the supported arguments - what they are, default values, dtype, etc.

Reply:
where should I add this description.

Reply:
You can add it above in this doc.